addons/aircraft/CfgAmmo.hpp

```diff
@@ -51,8 +51,25 @@ class CfgAmmo {
 
     // adjust damage and splash damage, closer to bluefor gatling with same caliber
     class Cannon_30mm_HE_Plane_CAS_02_F: Gatling_30mm_HE_Plane_CAS_01_F {
-        hit = 70; // default: 150
-        indirectHit = 11; // default: 4
+        hit = 65; // default: 150
+        indirectHit = 10; // default: 4
         indirectHitRange = 3; // default: 3
     };
+
+    // adjust damage and splash damage, AP Rounds
+    class Cannon_30mm_HE_Plane_CAS_03_F: Gatling_30mm_HE_Plane_CAS_01_F {
+        hit = 75; // default: 150
+        indirectHit = 1; // default: 4
+        indirectHitRange = 0.25; // default: 3
+        deflecting = 5;
+    };
+
+    class SubmunitionBase;
+    class ACE_30mm_mix: SubmunitionBase {
+        simulation = "shotSubmunitions";
+        submunitionAmmo[] = {"Gatling_30mm_HE_Plane_CAS_03_F",0.8,"Cannon_30mm_HE_Plane_CAS_01_F",0.2};
+        weaponType = "cannon";
+        model = "\A3\Weapons_f\empty";
+        triggerTime = 0.005;
```

Comment:
Other classes like this have considerably more config entries. For example:
```cpp
		class B_762x51_Minigun_Tracer_Red: SubmunitionBullet
		{
			hit = 11.6;
			indirectHit = 0;
			indirectHitRange = 0;
			model = "\A3\Weapons_f\Data\bullettracer\tracer_red";
			caliber = 3.6;
			deflecting = 25;
			explosive = 0;
			triggerTime = 0.1;
			submunitionConeAngle = 0;
			submunitionCount = 1;
			submunitionAmmo = "B_762x51_Minigun_Tracer_Red_splash";
			cartridge = "FxCartridge_762";
			visibleFire = 5;
			audibleFire = 70;
			cost = 1.2;
			airLock = 1;
			typicalSpeed = 800;
			tracerScale = 0.6;
			tracerStartTime = 0.0075;
			tracerEndTime = 5;
			airFriction = -0.001;
			class CamShakeExplode
			{
				power = 2.82843;
				duration = 0.6;
				frequency = 20;
				distance = 8.48528;
			};
			class CamShakeHit
			{
				power = 5;
				duration = 0.2;
				frequency = 20;
				distance = 1;
			};
		};
```
____

`subMunitionCount` may be able to have us simulate two or three projectiles from just a single shot, which would could make us reduce the rof to something that works in MP. Could be combined with that one entry that makes the ammo count show a multiple of the actually remaining rounds.

Reply:
Done and implemented for the 30mm cannon. Each bullet fired as a submunition is 3 rounds that spread out, and each time the cannon is fired, 3 bullets are removed from the magazine. To maintain 60 "rounds" per second, only ~20-22FPS are needed, instead of 60.